scene/3d/cpu_particles_3d.cpp

```diff
@@ -595,7 +595,7 @@ static real_t rand_from_seed(uint32_t &seed) {
 }
 
 void CPUParticles3D::_update_internal() {
-	if (particles.size() == 0 || !is_visible_in_tree()) {
+	if (particles.size() == 0 || !is_inside_tree()) {
```

Comment:
```suggestion
	if (particles.size() == 0 || !is_inside_tree() || !is_visible_in_tree()) {
```
I don't think we should remove the visibility check

Reply:
Done